infrastructure-playbooks/lv-create.yml

```diff
@@ -0,0 +1,58 @@
+- name: Creates Logical volumes for the Bucket Index or FS Journals on a single device. Prepares for use of osd_scenario=lvm. 
+  hosts:
+  - osds
+
+  tasks:
+
+  - name: Include vars of lv_vars.yaml
```

Comment:
Wy not just include all these directly in this playbook under a ``vars`` section?

Reply:
I see now that we're using the same variables in the ``lv-teardown.yml``, which is a strong reason for keeping this a file we include. We'll just need to make sure documentation is clear on usage and ensuring we can include ``lv_vars.yml`` correctly.